server/models/bill.coffee

```diff
@@ -13,6 +13,10 @@ module.exports = Bill = cozydb.getModel 'Bill',
     fileId: String
     content: String
     isRefund: Boolean
+    clientId: String # Client number in vendor CRM
```

Comment:
Ok, but I think we should distinguish the data we receive from the service and the data we store. They do not have to match one to one, in my opinion, and we can perform some changes on the data.

In this case, I think the user does not care about which number is associated to him in the SAP. He will most likely not even know what a SAP is. But he may have two contracts (let's say one for his primary house and one for his secondary house) and he wants to know which Bill belongs to which contract. Not sure about the data we get back and if it is doable, but performing changes on the data may be useful.

Same from previous models, if I receive some data from EDF and some data from Enercoop, I do not mind about the provider-specific fields, I just want to compare bills and know how much it costs from each provider.